CatAnalyzer/python/leptonSF_cff.py

```diff
@@ -442,7 +456,8 @@ def combineSF(set1, set2, additionalUnc1=0, additionalUnc2=0):
 
 ## Combined Scale factors
 ## id syst 1%+0.5%(quadrature), iso syst 1%
-muonSFTight = combineSF(muonSFTightIdOnly, muonSFTightIsoOnly, (0.01**2+0.005**2)**0.5, 0.01)
-muonSFTightGH = combineSF(muonSFTightGHIdOnly, muonSFTightGHIsoOnly, (0.01**2+0.005**2)**0.5, 0.01)
+muonSFTightBF = combineSF(combineSF(muonSFTightIdOnly, muonSFTightIsoOnly, (0.01**2+0.005**2)**0.5, 0.01), muonSFTrackingOnly)
+muonSFTightGH = combineSF(combineSF(muonSFTightGHIdOnly, muonSFTightGHIsoOnly, (0.01**2+0.005**2)**0.5, 0.01), muonSFTrackingGHOnly)
+muonSFTight = computeAverageSF(muonSFTightBF, 20.4, muonSFTightGH, 16.7)
```

Comment:
since the trackingSF starts from -2.4, the overall muon SF are given in eta_bins, not abseta_bins. this have to be propagated to the analysis modules. A fix is being prepared.